src/50_Samples_%26_Templates/Product_Page.html

```diff
@@ -403,16 +414,16 @@ <h2 class="m1">Apple</h2>
         <amp-img on="tap:AMP.setState({product: {selectedSlideForGreen: 0}})"
                   src="/img/green_apple_1_60x40.jpg"
                   width="60" height="40"
-                  class="selected"
-                  [class]="product.selectedSlideForGreen == 0 ? 'selected' : '' "
+                  class="selected animated fadeIn"
```

Comment:
I wouldn't fade the elements in initially (this doesn't work for carousels anyway).